### PR TITLE
Fix `hide-useless-comments` selectors

### PR DESCRIPTION
**Pull request by yakov116**
1. LINKED ISSUES:
   #3992 

2. TEST URLS:
   https://github.com/justinfrankel/licecap/issues/97

3. SCREENSHOT:
   Before
image
after
image

### CODE REVIEW

**Comment by fregante** [comment]
As long as it was tested on the licecap thread for the similar comment it’s good to be merged

**Comment by yakov116** [comment]
@cheap-glitch can you look this over? Why do we use different selectors for hide and unhide?

File: source/features/hide-useless-comments.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ function unhide(event: delegate.Event<MouseEvent, HTMLButtonElement>): void {
 	}
 
 	// Expand all "similar comments" boxes
-	for (const similarCommentsExpandButton of select.all('.js-discussion .Details-content--closed > span:only-child')) {
+	for (const similarCommentsExpandButton of select.all('.pagination-loader-container .Details-content--closed > span:only-child')) {
```
Comment (cheap-glitch) [inline source/features/hide-useless-comments.tsx:16]: @yakov116 we can indeed simplify this selector to `'.js-discussion .Details-element:not([data-body-version]) > summary'`

Comment (yakov116) [reply source/features/hide-useless-comments.tsx:16]: Thought so but it was waay past my bed time and I wanted to release it. (We don't want broken features)

I put it on my list for my next lint

**Comment by fregante** [comment]
This broke https://github.com/sindresorhus/refined-github/pull/3992

In the linked Licecap thread you can clearly see that the similar comment is hidden, but not expanded later:

Screen Shot

**Comment by fregante** [comment]
Also it might be a good time to also fix that little corner. The rule should be be:

```css
.rgh our class .timeline-comment--caret:before {
	border-right-color: var(--github-red);
}
```

**Comment by cheap-glitch** [comment]
> In the linked Licecap thread you can clearly see that the similar comment is hidden, but not expanded later

Weird, it's working for me (Firefox + `21.2.23`).

**Pull request merged**